### ISSUE TITLE
Enable setting limit per window (in a multi-window layout)

### ISSUE THREAD

**Issue by joaquinmoreira** [travmik/ZenTabs#11]
Is there a way to set the tabs limit per each window in a multi-window layour (e.g grid)?

**Comment by travmik** [comment]
Do you mean something like this?
![pndrp](https://f.cloud.github.com/assets/1381942/1212779/2d24bfc0-261e-11e3-8043-2d9cbe173cb4.jpg)

**Comment by joaquinmoreira** [comment]
No. I mean, for example in a grid layout, limit the amount of tabs per each of one the four windows of the layout. The current behaviour of zen tabs is limiting the global amount of opened tabs, no matter in what window is it opened, so my idea its to limit the amount per each window, e.g. a limit of 4 will make no more than 4 tabs can be opened in every window, thus keeping a maximum of 16.

**Comment by travmik** [comment]
Yes, I see. But what do you call "window"? I am using several windows in my workflow. But those windows are real windows and for each of them tabs are limited separately. 
You said "grid" and I thought that when you said window and grid you meant something like on image above. Doesn't you?

**Comment by joaquinmoreira** [comment]
Sorry for my mistake, i don't know the usual word for this, but i guess it should be called pane or something like that, when i erroneuosly used the word i meant the portion of the grid in which you can have multiple tabs.

**Comment by travmik** [comment]
Yeah - I think it calls panes. 
Actually I don't use them and don't know how to programatically manage them. But I like idea. I will add it to TODO and take a look at panes closer. 
Now I'm going to make new release with feature that I really like - favourite tabs. =) I will back soon to you with decision about you request. 

Will be in touch.

**Comment by littke** [comment]
I second this request.

**Comment by fascox** [comment]
me too, but it make sense only for horizontal splitted panes...
